test/windowclose.jl

```diff
@@ -11,6 +11,9 @@ window = GLFW.CreateWindow(800, 600, "InexactError")
 ccall( (:glfwSetWindowShouldClose, GLFW.lib), Void, (GLFW.WindowHandle, Cint), window, 189)
 @test GLFW.WindowShouldClose(window)
 
+sleep(0.25)
```

Comment:
Done. There's another `sleep` at the very end of the file; I left that one in place.